### ISSUE TITLE
Call to an undefined method Illuminate\Database\Query\Builder::firstOr()

### ISSUE THREAD

**Issue by mathieutu** [nunomaduro/larastan#114]
- Larastan Version: 0.2.4
- `--level` used: default

### Description:

  Line   app/Models/Model.php                                                       
 ------ --------------------------------------------------------------------------- 
  25     Call to an undefined method Illuminate\Database\Query\Builder::firstOr().  
 ------ --------------------------------------------------------------------------- 

### Laravel code where the issue was found:
 ```php
return $this->query()->firstOr(function () {
            return 'foo';
        });
```

**Comment by hartman** [comment]
Is that really the code where it was found ? Because the function signature is:
`public function firstOr($columns = ['*'], Closure $callback = null)`

so....

**Comment by hartman** [comment]
Ah duh. Anyway, does explain why the analyser fails on it. Will probably have to be added to the whitelist

**Comment by nunomaduro** [comment]
Because this package don't have a stable version yet. And this is probably
a false positive. I will check it more carefully after my holidays.

**Comment by nunomaduro** [comment]
@mathieutu This issue got addressed on the version `v0.2.11`. Can you test it?

**Comment by nunomaduro** [comment]
This should be fixed.
